src/main/java/com/example/doodle_clone/controller/UserAccountController.java

```diff
@@ -0,0 +1,99 @@
+package com.example.doodle_clone.controller;
+
+import jakarta.mail.SendFailedException;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.stereotype.Controller;
+import com.example.doodle_clone.models.user.ConfirmationToken;
+import com.example.doodle_clone.models.user.User;
+import com.example.doodle_clone.repo.ConfirmationTokenRepository;
+import com.example.doodle_clone.repo.UserRepository;
+import com.example.doodle_clone.services.EmailSenderService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.ModelAndView;
+
+import java.util.List;
+
+@Controller
+@Getter@Setter
+public class UserAccountController {
+    @Autowired
+    private UserRepository userRepository;
+
+    @Autowired
+    private ConfirmationTokenRepository confirmationTokenRepository;
+
+    @Autowired
+    private EmailSenderService emailSenderService;
+
+    @RequestMapping(value="/register", method = RequestMethod.GET)
+    public ModelAndView displayRegistration(ModelAndView modelAndView, User user)
+    {
+        modelAndView.addObject("user", user);
+        modelAndView.setViewName("register");
+        return modelAndView;
+    }
+
+    @RequestMapping(value="/register", method = RequestMethod.POST)
+    public ModelAndView registerUser(ModelAndView modelAndView, User user) throws SendFailedException
+    {
+
+        User existingUser = userRepository.findByEmailId(user.getEmailId());
+        if(existingUser != null)
+        {
+            modelAndView.addObject("message","Пошта зайнята!");
+            modelAndView.setViewName("error");
+        }
+        else
```

Comment:
we don't format code like that 
} else {
}

Reply:
fix